src/lib/FreeC/Backend/Coq/Converter/Expr.hs

```diff
@@ -40,6 +40,15 @@ convertExpr expr = convertExpr' expr [] []
 convertExpr' :: IR.Expr -> [IR.Type] -> [IR.Expr] -> Converter Coq.Term
 
 -- Constructors.
+-- 
+-- Partial applied constructors are not evaluated in Haskell and therefor
+-- cannot be @⊥@. The translated type of a constructor @C : τ₀ -> … -> τₙ@ is
+-- @c : τ₀' -> … -> τₙ*@ instead of @m(τ₀' -> m(τ₁' -> m(… -> τₙ')))@.
```

Comment:
Unfortunately, I see here some undefined (unicode?) characters : /

Reply:
The [styleguide](https://github.com/FreeProving/guidelines/blob/master/CONTRIBUTING.md#haddock-styleguide) suggested to use Unicode symbols over inline LaTeX. I thought the comment would be easier to read with constructor parameters that use actual subscript indices (similar to the algorithm description in [DecreasingArguments.hs](https://github.com/FreeProving/free-compiler/blob/master/src/lib/FreeC/Backend/Coq/Analysis/DecreasingArguments.hs)). The comment contains a Greek `tau`, subscript `1`, `2` and `n` and the `bottom` symbol.

Reply:
It's the _subscript n_ that my browser refuses to render : / Weird, but it's finde then.